client/cucumber/features/upgrade.feature

```diff
@@ -13,7 +13,7 @@ Feature: Upgrade Smart Contracts
         And the contract has been packaged
         And the package has been installed
         When I upgrade the installed package with the transaction '' and args '', not using private data
-        Then there should be a instantiated smart contract tree item with a label '<upgradedName>' in the 'Local Fabric Ops' panel
+        Then there should be a instantiated smart contract tree item with a label '<upgradedName>' in the 'Fabric Environments' panel
```

Comment:
For grammatical purposes, we should make this:
_"Then there should be a**n** instantiated..."_